### ISSUE TITLE
Different results between using python 3 and using python 2

### ISSUE THREAD

**Issue by fehiepsi** [pytroll/aggdraw#14]
It seems that there are some differences in implementation of python 3. Here is a simple script to display the issue. In python 2, the result is `50800`. In python 3, the print result is `0`.
```
import aggdraw
from PIL import Image
import numpy as np

symbol = aggdraw.Symbol("M400 200 L400 400")
pen = aggdraw.Pen("red")
image = Image.fromarray(np.zeros((800, 600, 3)), mode="RGB")
canvas = aggdraw.Draw(image)
canvas.symbol((0, 0), symbol, pen)
canvas.flush()
print(np.asarray(image).sum())
```

**Comment by davidh-ssec** [comment]
Thanks for the report. I assume you installed aggdraw from this master branch? Python 3 compatibility was only recently added and hasn't even been released yet so it's possible we missed something. Could you try a couple things for me to help narrow down the problem?

1. Save the image to disk.
2. Open the image in your system's image viewer, does it contain what you expect?
3. Open the image from disk in python and try summing it.

What version of python 3 are you using?

**Comment by fehiepsi** [comment]
Hi @davidh-ssec , thanks for your reply. I installed aggdraw from this master branch. I have saved images into disk and opened it in image viewer. For python 2, we have a red line on black background, which is expected. For python 3, we have a fully black image (of course, the sum is `0` suggests output image will be fully black). This problem appears in all python 3.3, python 3.4, python 3.5, python 3.6 in my system.

**Comment by davidh-ssec** [comment]
@fehiepsi What operating system are you using? If I have time I will test this tomorrow.

**Comment by fehiepsi** [comment]
Thank you @davidh-ssec, I use Ubuntu 16.04.1 64 bits.

**Comment by mraspaud** [comment]
@davidh-ssec I can reproduce the problem on python 3.6. Works fine in 2.7

**Comment by howff** [comment]
I just downloaded master and tried it on Python 2.7.6 on ubuntu 14.04. I get the result 0 (not 50800) so the test fails, but this conflicts with what the original poster reported about Python 2.

**Comment by davidh-ssec** [comment]
@howff Could you make sure you are running the code you think you are running `print(aggdraw.__file__)`? The PR mentioned here (#15) ran travis tests on Ubuntu 14.04 on Python 2.7.13 and passed: https://travis-ci.org/pytroll/aggdraw/builds/285638918

What's changed?

**Comment by howff** [comment]
There's no way to get aggdraw without this repo (unless I add ppa:chris-lea/python-aggdraw which I haven't done) so there can't be multiple copies. I just downloaded the zip, unpacked, ran python setup.py build_ext -i (not in a virtualenv), ran python selftest.py and it failed with assert np.asarray(image).sum() == 50800. In order to print(aggdraw.__file__) I modified selftest.py to import aggdraw as aggdraw, and got the path to aggdraw.so which has just been compiled. So I don't know what's changed.
